packages/cozy-sharing/src/components/recipient.styl

```diff
@@ -2,6 +2,10 @@
 @require 'settings/breakpoints.styl'
 @require 'settings/palette.styl'
 
+:root
+  --genericRecipientBackground: var(--silver)
+  --genericRecipientColor: var(--charcoalGrey)
```

Comment:
Not still very convinced by that. but since its better than the first version, lets use that